Source/MindTheWord.js

```diff
@@ -120,14 +121,14 @@ function length(obj) {
 
 function intersect() {
   var i,
-      all,
-      shortest,
-      nShortest,
-      n,
-      len,
-      ret = [],
-      obj = {},
-      nOthers;
+    all,
```

Comment:
It doesn't say specifically about variable declaration. But it does say the following for objects.

> Multiline array initializers and object initializers are indented 2 spaces, with the braces on their own line, just like blocks

```
var inset = {
  top: 10,
  right: 20,
  bottom: 15,
  left: 12
};
```

To maintain consistency, I believe we should stick to 2 space indentations for multiple variable declaration.